WidgetExtension/Up Next/EpisodeView.swift

```diff
@@ -62,27 +55,57 @@ struct EpisodeView: View {
     }
 }
 
-struct WidgetPlayToggleStyle: ToggleStyle {
-    func makeBody(configuration: Configuration) -> some View {
-        ZStack {
-            configuration.label
-                .truncationMode(.tail)
+struct WidgetFirstEpisodePlayToggleStyle: ToggleStyle {
+    let colorScheme: PCWidgetColorScheme
 
-            Circle()
-                .foregroundStyle(.white)
-                .frame(width: 24, height: 24)
+    func makeBody(configuration: Configuration) -> some View {
+        HStack(spacing: 0) {
             Group {
                 configuration.isOn ?
                 Image("icon-pause")
                     .resizable()
-                    .foregroundStyle(.black)
+                    .foregroundStyle(colorScheme.topButtonTextColor)
                 :
-                    Image("icon-play")
+                Image("icon-play")
                     .resizable()
-                    .foregroundStyle(.black)
+                    .foregroundStyle(colorScheme.topButtonTextColor)
+            }
+            .frame(width: 28, height: 28)
+            // TODO: Something fun - create a timeline that counts down by the minute instead of showing "now playing"
```

Comment:
I just noticed I left this here... it is more of a "this would be fun if we could do this" activity, but not something that can easily be done right now.

Comment should probably be removed, but I'll leave it here for the moment incase anyone has thoughts on it.

Reply:
Instead of the TODO comment add an issue on the repo as an improvement/suggestion to do in the future.